sashimi/processes/camera.py

```diff
@@ -123,13 +123,17 @@ def __init__(
         self.was_waiting = False
 
     def initialize_camera(self):
-        if conf["scopeless"]:
-            self.camera = camera_class_dict["mock"]()
-        else:
-            self.camera = camera_class_dict[conf["camera"]["name"]](
-                camera_id=conf["camera"]["id"],
-                max_sensor_resolution=tuple(conf["camera"]["max_sensor_resolution"]),
-            )
+        camera_conf = conf.pop("camera")
+        camera_name = camera_conf.pop("name")
+
+        self.camera = camera_class_dict[camera_name](**camera_conf)
+
+        # if conf["scopeless"]:
```

Comment:
I don't understand how this will work. If one calls `--scopeless` then what gets set to mock? Unless calling scopeless will actually modify the config file __at entry__ so no `if conf["scopeless"]:` is needed to be wandering around the code as this would propagate by itself in the config file. Is this what you are looking for?

Reply:
Sorry, this is still around but the idea for this PR is that the default config will have mock in everything, and then people customise it and add stuff. For going back to the original mock modality on a configured setup, I will just have a quick way of regenerating the dict with the mock keys to be used only for a demo run.